src/features/app/App.tsx

```diff
@@ -7,9 +7,13 @@ import Routes from '../../routes'
 import Modal from './modal/Modal'
 import { wsConnect } from '../websocket/state/websocketActions';
 
-import './App.css';
 import SnackBar from '../snackBar/SnackBar';
 
+import './App.css';
+// import styles for react-responsive-carousel, used in workflow splash modal
+import 'react-responsive-carousel/lib/styles/carousel.min.css'

```

Comment:
move this import into `src/features/modal/SplashModal.tsx`. CSS imports should be colocated with the component that utilizes them